packages/codebytes/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../../tsconfig.json",
+  "include": ["../../typings/*.d.ts", "./src/**/*.ts", "./src/**/*.tsx"],
+  "compilerOptions": {
```

Comment:
do we need a ` "rootDir": "src/"` property here? @JoshuaKGoldberg

Reply:
thanks @aionate0812, good call out. i'll bring this up with josh when we pair!

Reply:
If it works without, you can remove it 😄 

`rootDir` is by default calculated as the longest common prefix for all included files. If that ends up being `src/` here then great.

https://aka.ms/tsconfig-reference